subsystems/hangSubsystem.py

```diff
@@ -11,23 +11,37 @@ class HangSubsystem(commands2.Subsystem):
     def __init__(self):
         super().__init__()
         self.hangMotor1 = rev.CANSparkMax(ELEC.hang_motor_CAN_ID, rev.CANSparkMax.MotorType.kBrushless)
-        '''self.hangMotor2 = rev.CANSparkMax(ELEC.lower_motor_CAN_ID, rev.CANSparkMax.MotorType.kBrushed)
+        self.hangMotor2 = rev.CANSparkMax(ELEC.lower_motor_CAN_ID, rev.CANSparkMax.MotorType.kBrushless)
 
-        self.hangMotor1 = phoenix5.TalonSRX(ELEC.hang_motor_CAN_ID)
+        '''self.hangMotor1 = phoenix5.TalonSRX(ELEC.hang_motor_CAN_ID)
```

Comment:
Don't leave strings (which are pretending to be comments) doing nothing in code that will be checked in.  This one looks safe to delete.